catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStoreImpl.java

```diff
@@ -473,4 +460,36 @@ public String getRegistryId() {
         return registryId;
     }
 
+    private class RegistryAvailabilityCommand implements AvailabilityCommand {
+
+        @Override
+        public boolean isAvailable() {
+            LOGGER.debug("Checking availability for source {} ", cswSourceConfiguration.getId());
+            boolean oldAvailability = RegistryStoreImpl.this.isAvailable();
+            boolean newAvailability;
+            // Simple "ping" to ensure the source is responding
+            newAvailability = (getCapabilities() != null);
+            if (oldAvailability != newAvailability) {
+                // If the source becomes available, configure it.
+                if (newAvailability) {
+                    configureCswSource();
+                    try {
+                        // Make sure the endpoint supports registry operations
+                        newAvailability = registryInfoQuery();
+                        if(!newAvailability){
+                            LOGGER.info("Unable to retrieve identity node of remote registry {}",
```

Comment:
❓  since these are at `info` level, how often do you think these log statements will be hit at runtime? Is there something an administrator can do to help resolve this?